test/test_static_runtime.py

```diff
@@ -195,6 +195,13 @@ def test_trivial_graph(self):
         o_test = tg_a(s, s, s)[0]
         torch.testing.assert_allclose(o_ref, o_test)
 
+    def test_leaky_relu(self):
+        s = torch.randn(5, 5)
+        tg = torch.jit.script(nn.LeakyReLU(0.1))
```

Comment:
directly scripting a builtin module is clever -- I wonder if `torch.fx` could be used with this paradigm to accelerate builtin modules with StaticRuntime when appropriate